src/tokenizer/index.js

```diff
@@ -631,10 +631,27 @@ export default class Tokenizer extends LocationParser {
   }
 
   readRadixNumber(radix: number): void {
+    const start = this.state.pos;
+    let isBigInt = false;
+
     this.state.pos += 2; // 0x
     const val = this.readInt(radix);
     if (val == null) this.raise(this.state.start + 2, "Expected number in radix " + radix);
+
+    if (this.hasPlugin("bigInt")) {
+      if (this.input.charCodeAt(this.state.pos) === 110) { // 'n'
```

Comment:
How about using `0x6E` instead of `110`? It makes it more obvious that this refers to U+006E LATIN SMALL LETTER N.